wemake_python_styleguide/visitors/tokenize/comments.py

```diff
@@ -60,9 +71,18 @@ def visit_comment(self, token: tokenize.TokenInfo) -> None:
             WrongMagicCommentViolation
 
         """
+        self._line_num = token.start[0]
```

Comment:
We try to keep the logic out of `visit_` methods

Reply:
Sure, that's reasonable. The intention was to put that in a helper function, but adding one more function makes this class exceed the default limit of number of functions/methods in one class. If we put it back into ``_check_empty_comment``, it would violate this rule (WPS231), exceeding the limit of code complexity. What is your suggestion?

Reply:
Let's create a new visitor than! 👍

Reply:
Sounds good! Just making sure, this means we will make a new class within ``comments.py`` and we will call it like ``EmptyCommentVisitor()``? Thanks

Reply:
Yes 👍